react/modules/interpolateLink.ts

```diff
@@ -16,8 +16,9 @@ export default function interpolateLink(params: Params) {
   let resolvedLink = link
 
   for (const key of Object.keys(variables)) {
+    const newKey = key.replace(/[-+%]/g,'')
```

Comment:
If you want to ignore all special characters the regex should be something like this `/[^\.]/g`

Reply:
But using that regex would replace almost everything, alternatively, a generic solution that allows us to handle special characters, removing them for precision, could be this one /[^\d\w.]/g.

Reply:
If you agree with this solution then I will commit it and push.

Reply:
Maybe a better approach can be, this interporlateLink receive a new param called `rejectionRegex` or `escapeLinkRegex` or something like that. By doing this the function will be more generic and won't break other users that want these special characters.
Example:
```tsx
interface Params {
  ...
  escapeLinkRegex?: Regex
}

function interpolateLink(params: Params) {
  ...
  for (const key ...) {
    let newKey = key
    if (params.escapeLinkRegex) {
      newKey = key.replace(params.escapeLinkRegex, '')
    } 
    ...
  }
}
```

PS.: This code is just an example

Reply:
I like the idea of receiving a RegEx parameter. But we would still need a default value... @anto90fg could you please go to https://regex101.com, input your regex and add a few test strings just so we can see it working and test it?

Reply:
The committed regex allows you to remove only the special characters in the key that trigger the error in the creation of the regex, the result in the image below.
![image](https://user-images.githubusercontent.com/73878310/107022014-26041a80-67a5-11eb-90d0-34258baf7fe3.png)
While the second one is more generic and allows to remove all special characters and spaces.
![image](https://user-images.githubusercontent.com/73878310/107021736-d1f93600-67a4-11eb-8881-7ae5003b9ac8.png)

Reply:
Nice @anto90fg. What do you think about @victorhmp suggestion, receiving a param with a RegEx?